### DISCUSSION TITLE
Authorize Relationship managers?

### DISCUSSION THREAD

**Discussion by SkeyPunyapal** [filamentphp/filament#4033] [Help] [answered]
do we have anything to prevent/allow particular users to access the whole Relationship manager or some part of it?

one more, do we have any option to give labels to widgets?

**Comment by danharrin** [comment, marked as answer]
Use the viewAny() method in the related model policy

And widgets can have headings by overriding getHeading() I think

**Reply by SkeyPunyapal** [reply]
I'm already using that. but I want to do something like someone can access blogs but cannot access the comments which are implemented with relationship managers.

thanks for the Widget tip I'll try it.

// getTableHeading() worked for me...

**Reply by danharrin** [reply]
You can use the CommentPolicy then, as well as the BlogPolicy?
